### PR TITLE
Matthew branch

### PR DESCRIPTION
**Pull request by michaelc445**
add character animations and sprites to single player and add sprites to multiplayer

### CODE REVIEW

**Review by Konrad-Dagiel — APPROVED** [review]
had to update nomad and fix jump attack animation
works perfect now

File: game/main.py [hunk 1]
```diff
@@ -515,6 +545,167 @@ def audio():
...
+    global p1
+    global p2
+
+    p1_color_wizard= "#d7fcd4"
+    p1_color_warrior= "#d7fcd4"
+    p1_color_nomad= "#d7fcd4"
+
+    p2_color_wizard= "#d7fcd4"
+    p2_color_warrior= "#d7fcd4"
+    p2_color_nomad= "#d7fcd4"
```
Comment (michaelc445) [inline game/main.py:614]: dont think we need all these

File: game/main.py [hunk 2]
```diff
@@ -515,6 +545,167 @@ def audio():
...
+                if p2_warrior.checkForInput(mouse):
+                    p2_color_warrior = "Blue"
+                    p2_color_nomad = "#d7fcd4"
+                    p2_color_wizard = "#d7fcd4"
+                    p2="warrior"
+                if p2_nomad.checkForInput(mouse):
+                    p2_color_nomad = "Blue"
+                    p2_color_wizard = "#d7fcd4"
+                    p2_color_warrior = "#d7fcd4"
+                    p2="nomad"
```
Comment (michaelc445) [inline game/main.py:704]: can we use a loop for this instead?

File: game/online_fighter.py [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+    #                 self.attack_type = 2
+    #
+    #             self.attack(surface, target)
+    #
+    #     # block
+    #     if key[player_controls["block"]]:
+    #         self.color = (0, 0, 255)
+    #         self.blocking = True
+    #     else:
+    #         self.blocking = False
```
Comment (michaelc445) [inline game/online_fighter.py:119]: remove commented out code

**Review by dylanOmahony2000 — APPROVED** [review]
all looks good

**Pull request merged**